Makefile

```diff
@@ -18,6 +18,8 @@ default: compile
 all: compile build build-startup-script build-executable-script build-test-script
 
 compile:
+	git submodule update --init --recursive
```

Comment:
I just want to confirm one more time that there's a somewhat strong reason to include templates as a git submodule, since it would be a lot simpler to put those templates in the repo directly. There's some complexity cost in using a submodule

Reply:
The `kbase_report_templates` repo is meant to be usable locally by app developers -- there's a little script for them to test out templates whilst they're developing them.
Perl apps can also use the report templates by adding the `kbase_report_templates` repo directly to their apps (I've already written a module for template rendering in some work I did for Chris), rather than having to pull in a py repo. There may also be some further development on the `kbase_report_templates` and it's much easier for me to monitor that stuff and then submit a PR to get the submodule updated in the KBaseReport repo than you having to approve PRs for template alterations.

Reply:
Yeah, definitely makes sense about the PR process